src/main/resources/version.properties

```diff
@@ -0,0 +1 @@
+version = ${project.version}
```

Comment:
This pulls data from the Maven artifact or?

Reply:
Yes, I followed this tutorial https://www.hackingnote.com/en/maven/load-pom-properties-in-java/